rpcs3/Emu/Cell/Modules/cellPad.cpp

```diff
@@ -972,6 +986,27 @@ error_code cellPadLddUnregisterController(s32 handle)
 	return CELL_OK;
 }
 
+error_code cellPadFilterIIRInit(vm::ptr<CellPadFilterIIRSos> pSos, s32 cutoff)
+{
+	sys_io.todo("cellPadFilterIIRInit(pSos=*0x%x, cutoff=%d)", pSos, cutoff);
+
+	if (!pSos) // TODO: does this check for cutoff > 2 ?
+	{
+		return CELL_PADFILTER_ERROR_INVALID_PARAMETER;
+	}
+
+	return CELL_OK;
+}
+
+u32 cellPadFilterIIRFilter(vm::ptr<CellPadFilterIIRSos> pSos, u32 filterIn)
+{
+	sys_io.todo("cellPadFilterIIRFilter(pSos=*0x%x, filterIn=%d)", pSos, filterIn);
+
+	// TODO: apply filter
+
+	return std::clamp(filterIn, 0u, 1023u);
```

Comment:
```suggestion
	return std::clamp<u32>(filterIn, 0, 1023);
```

Reply:
Is this in any way different apart from how it looks?

Reply:
Its clearly more readable and doesn't look like a compilation issue workaround.

Reply:
0 and 32 are passed as int though, whereas 0u and 1023u are clearly u32

Reply:
yours looks more like a compilation workaround tbh

Reply:
by that logic everything in cpp is a compilation workaround